CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
 
 ##### Enhancements
 
+* Exclude the local spec-repos directory from Time Machine Backups.
```

Comment:
please add two empty spaces at the end of this sentence for markdown formatting.